pype/lib.py

```diff
@@ -746,8 +746,9 @@ def execute(self, *args, **kwargs):
 
 def get_linked_assets(asset_entity):
     """Return linked assets for `asset_entity`."""
-    # TODO implement
-    return []
+    inputs = asset_entity["data"].get("inputs", [])
+    inputs = [io.find_one({"_id": x}) for x in inputs]
```

Comment:
I still don't agree with key `"inputs"`, it says nothing about it's content.

About the code, I think it should be rather:
```
input_ids = asset_entity["data"].get("inputs") or []
return list(io.find({"_id": {"$in": input_ids}}))
```

Querying documents by one is slow as hell torture.

Reply:
> I still don't agree with key "inputs", it says nothing about it's content.

Think this will need to be taken up internally in Pype.

> Querying documents by one is slow as hell torture.

Will need to be another PR, but maybe we need to hit the slowness first?

Reply:
we all can agree the name should be improved in future to something like `links` as @mkolar had said. But for now lets stick to the upstream consensus.. https://github.com/pypeclub/pype/pull/475#discussion_r474823179